### DISCUSSION TITLE
Controlling snapclients with homeassistant or node red

### DISCUSSION THREAD

**Discussion by Sungod3000** [snapcast/snapcast#1072] [Q&A]
Hi, 

I was wondering how I would go about controlling which clients play at any time with homeassistant?

I managed to get everything running with mopidy and snapcast and I have clients in 3 rooms. 

Since I mainly play in one room it would be a bit overkill to constantly play everywhere. So I wonder how that (or if) would be controlled?

Cheers.
F.

**Comment by GiODi20** [comment]
https://www.home-assistant.io/integrations/snapcast/

**Reply by Sungod3000** [reply]
yes, i saw that and put it in my configuration. 
do you know a little more advanced tutorial about integrating it further?

**Reply by Sungod3000** [reply]
Alright, made some progress. The snapcast intergration works now and I can control clients via the media player service. 

I can call music directly and choose the clients on which it should play, but how can I redirect a client when theres already music playing?
Something to switch the existing client to another one?
